x-pack/plugins/spaces/public/management/edit_space/edit_space_general_tab.tsx

```diff
@@ -137,7 +143,7 @@ export const EditSpaceSettingsTab: React.FC<Props> = ({ space, features, history
       setIsLoading(true);
 
       let disabledFeatures: string[] | undefined;
-      if (spaceClone.solution === SOLUTION_VIEW_CLASSIC) {
+      if (spaceClone.solution == null || spaceClone.solution === SOLUTION_VIEW_CLASSIC) {
```

Comment:
```suggestion
      if (!spaceClone.solution || spaceClone.solution === SOLUTION_VIEW_CLASSIC) {
```
might be better? On the API side that field is `undefined`

Reply:
@elena-shostak `== null` checks for both `undefined` and `null` value

Reply:
yeah, but if we are still doing type conversion we can probably just do `!spaceClone.solution`. That case `spaceClone.solution == null` won't handle empty string for example (we don't have such, but still).

Reply:
@elena-shostak Thanks! I'm accepting your suggestion so it'll be more readable.